src/plugins/data/public/antlr/dql/grammar/.antlr/DQLLexer.interp

```diff
@@ -0,0 +1,56 @@
+token literal names:
```

Comment:
we should be able to remove these.

Reply:
will remove grammar/.antlr in fast follow

Reply:
Do we still want to remove these in this PR? should be a fast removal

Reply:
since this change would be based on your pr, it would be best to handle it in a fast follow as to not remove actual functionality by accident. basically, i want keep changes separate so that there will be a working version before modifying again.